go.mod

```diff
@@ -4,6 +4,7 @@ go 1.21
 
 require (
 	github.com/Masterminds/sprig/v3 v3.2.2
+	github.com/Mellanox/sriovnet v1.0.3
 	github.com/blang/semver v3.5.1+incompatible
```

Comment:
nit: please open an issue in the operator repo I would like to see this package moving under the npwg org if possible

Reply:
It is already part of the NPWG. I updated code to use the right version